test/integration/contracts/mixins/with-events.spec.ts

```diff
@@ -92,8 +101,8 @@ beforeAll(async () => {
 	});
 });
 
-afterAll(() => {
-	return testUtils.destroyContext(ctx);
+afterAll(async () => {
+	await testUtils.destroyContext(ctx);
```

Comment:
We don't need `async`/`await` here, we can just return `testUtils.destroyContext(ctx);` and have the same result.